functions/first-login.ts

```diff
@@ -52,7 +52,7 @@ export async function handlePostRequest(
 		properties.name = formData.get('name') || ''
 	}
 
-	await createPerson(domain, db, userKEK, payload.email, properties)
+	await createPerson(domain, db, userKEK, payload.email, properties, true)
```

Comment:
@xtuc with this we set the user you use for the first-login as an admin, what do you think?

Reply:
This will set admin to all users; first-login triggers for all users on their first login.

Reply:
ahhhh, I thought it was just the first log in for the whole instance, sorry my bad 😓